pkg/printers/internalversion/printers.go

```diff
@@ -831,6 +836,10 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 				reason = "NotReady"
 			}
 		}
+		// change pod status back to "Error" if there is at least one container still reporting as "Error" status
+		if hasError {
```

Comment:
this block changes the original logic, see Line 831, for example what the reason should be while there is a container running and a container exits error?

Reply:
That's a good question.  
Line 831 is show while reason is 'Completed', It should be reported to Running because it exit sucessfully.
But while a container terminated unsuccessfully, the pod status should be chang to error.